site/docs/plugins/commands.md

```diff
@@ -5,10 +5,41 @@ next: false
 
 # Commands (`commands`)
 
-Coming soon, please come back later.
+Command handling on steroids.
+
+This plugin provides various features related to command handling that are not contained in the [command handling done by the grammY core library](../guide/commands).
+Here is a quick overview of what you get with this plugin:
+
+- Better code structure by attaching middleware to command definitions
+- Automatic synchronization via `setMyCommands`
+- Defining command translations
+- Command scope handling
+- "Did you mean ...?" feature that automatically finds the nearest existing command
+
+All of these features are made possible because you will define one or more central command structures that you define for your commands.
```

Comment:
```suggestion
All of these features are made possible because you will define one or more central command structures that define your bot's commands.
```